src/table/cells/__tests__/handle-key-press.spec.js

```diff
@@ -235,5 +251,99 @@ describe('handle-key-press', () => {
       expect(evt.stopPropagation).to.have.been.calledOnce;
       expect(changeSortOrder).to.have.been.calledOnce;
     });
+
+    it('when press ArrowRight and shif and ctrl key, should not update the sorting', () => {
+      evt.key = 'ArrowRight';
+      evt.shiftKey = true;
+      evt.ctrlKey = true;
+      headHandleKeyPress(evt, rootElement, rowIndex, colIndex, changeSortOrder, layout, isDim);
+      expect(evt.preventDefault).to.not.have.been.calledOnce;
+      expect(evt.stopPropagation).to.not.have.been.calledOnce;
+      expect(changeSortOrder).to.not.have.been.calledOnce;
+    });
+  });
+
+  describe('updatePage', () => {
+    const sandbox = sinon.createSandbox();
+    let evt = {};
+    let totalRowSize;
+    let page;
+    let rowsPerPage;
+    let handleChangePage;
+
+    beforeEach(() => {
+      evt = {
+        shiftKey: true,
+        ctrlKey: true,
+        metaKey: true,
+        key: 'ArrowRight',
+      };
+      handleChangePage = sandbox.spy();
+    });
+
+    afterEach(() => {
+      sandbox.verifyAndRestore();
+    });
+
+    it('when shift key is not pressed, handleChangePage should not run', () => {
+      evt.shiftKey = false;
+      updatePage(evt, totalRowSize, page, rowsPerPage, handleChangePage);
+      expect(handleChangePage).to.not.have.been.calledOnce;
+    });
+
+    it('when ctrl key or meta key is not pressed, handleChangePage should not run', () => {
+      evt.ctrlKey = false;
+      evt.metaKey = false;
+      updatePage(evt, totalRowSize, page, rowsPerPage, handleChangePage);
+      expect(handleChangePage).to.not.have.been.calledOnce;
+    });
+
+    it('handleChangePage should not run', () => {
+      page = 0;
+      totalRowSize = 40;
+      rowsPerPage = 40;
+      updatePage(evt, totalRowSize, page, rowsPerPage, handleChangePage);
+      expect(handleChangePage).to.not.have.been.calledOnce;
+    });
+
+    it('handleChangePage should not run', () => {
+      evt.key = 'ArrowLeft';
+      page = 0;
+      updatePage(evt, totalRowSize, page, rowsPerPage, handleChangePage);
+      expect(handleChangePage).to.not.have.been.calledOnce;
+    });
```

Comment:
These two need unique names, describing why they should not run. Same thing for should change page below.
Also I think all arguments should be defined, totalRowSize and rowsPerPage are missing for the bottom one